integration-tests/test_client.py

```diff
@@ -55,9 +84,18 @@ def test_client_rpm_mandatory_files(filename, rpm_ql_insights_client):
 
 @pytest.mark.usefixtures("register_subman")
 def test_client_logfiles_mask(insights_client):
-    """Verify that files in /var/log/insights-client have the right mode: 0600
-
-    Ref: https://bugzilla.redhat.com/show_bug.cgi?id=1955724
+    """
+    :id: 8f24500c-d0ff-4ab1-a2ae-3b99cbf68e36
+    :title: Verify Client logfiles Mask
```

Comment:
this is not about a file/permission mask (like umask), but rather about the permissions themselves